helix/principles/sitecore-items/development.rst

```diff
@@ -45,75 +43,124 @@ serialization comes in handy.
 
 Serialization allows items in the databases (typically in the Master and
 Core database) to be written to disk in a text-based format and
-subsequently restored into a database.
+subsequently restored into a database. Sitecore provides the
+`Team Development for Sitecore (TDS) <https://www.teamdevelopmentforsitecore.com/TDS-Classic>`__
```

Comment:
Might be worth checking with Charlie, I think this might now be called "Sitecore TDS"?

Reply:
@kamsar is correct, the new name is Sitecore TDS.

Reply:
Fixed all TDS references, good catch